test/e2e/utils/utils.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 Intel Corporation. All Rights Reserved.
+// Copyright 2020-2022 Intel Corporation. All Rights Reserved.
```

Comment:
What about the copyright years of the files below?
Do we change?
- cmd/fpga_admissionwebhook/main.go
- cmd/sgx_admissionwebhook/main.go
- test/e2e/sgxadmissionwebhook/sgxaadmissionwebhook.go

Reply:
fixed